lib/ex_unit/lib/ex_unit.ex

```diff
@@ -270,6 +270,9 @@ defmodule ExUnit do
       prints each test case and test while running. Note that in trace mode test timeouts
       will be ignored.
 
+  Any arbitrary configuration can also be passed to `configure/1`, and these options
```

Comment:
I think you can also pass the configuration options to `start/1`, should that be mentioned here?